### DISCUSSION TITLE
当回答包含“$”时，会被吃掉

### DISCUSSION THREAD

**Discussion by mwell953** [zai-org/ChatGLM3#684] [Q&A] [answered]
“$”这个符号的生成会被吃掉怎么办啊，很急！！！！！

https://github.com/THUDM/ChatGLM3/assets/155444324/de8a1030-3e5e-4714-b335-e6d1076bea51

**Comment by mwell953** [comment]
<img width="805" alt="屏幕截图 2024-01-02 195417" src="https://github.com/THUDM/ChatGLM3/assets/155444324/24967a51-2c49-4089-9370-cea0ffbd8357">
提问时也输入了“$”，同样被吃掉了。
我部署的是Chatglm3-6B int13的同样也是这个效果
<img width="829" alt="屏幕截图 2024-01-02 195439" src="https://github.com/THUDM/ChatGLM3/assets/155444324/54262149-0f09-4522-b4ee-7c07370d8333">

**Comment by zRzRzRzRzRzRzR** [comment]
感觉是因为streamlit的问题，命令行行吗

**Reply by mwell953** [reply]
命令行？啥意思啊

**Reply by mwell953** [reply]
就是这种错误：⚠formula incomplete。

**Reply by mwell953** [reply]
确实是streamlit的问题，我用cli_demo.py试了一下，是没有问题的，但是放到streamlit上就会被吃掉，请问怎么解决啊，能否给一点思路。谢谢

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
streamlit 的问题我们解决不了的，这是steramlit库的问题
